cluster/saltbase/salt/docker/docker.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket
+Requires=docker.socket
+
+[Service]
+EnvironmentFile={{ environment_file }}
+ExecStart=/usr/bin/docker -d -H fd:// "$DOCKER_OPTS"
+MountFlags=slave
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=infinity
+Restart=always
+RestartSec=2s
+StartLimitInterval=0
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
@justinsb - So the behavior differences are not major concern right now for the Vagrant setup, I think I am fine if you keep the file you have now.  I am not running with any custom storage solutions now, so I don't think I need the ease of the separate environment files.